web/chromecom.js

```diff
@@ -275,7 +275,7 @@ var ChromeCom = (function ChromeComClosure() {
     function onDisconnect() {
       // When the connection fails, ignore the error and call the callback.
       port = null;
-      onCompleted();
+      callback();
```

Comment:
I might be missing something obvious here, but won't this cause the `eventListeners` to remain attached? Or, does that simply not matter here?